.validate-branch-namerc.js

```diff
@@ -0,0 +1,11 @@
+const BRANCH_REGEX =
```

Comment:
Also, going along with the other suggestion above, could this also be a TypeScript file? If it's too much trouble because it requires monkeying with the TypeScript/ESLint config, no problem, but I thought I'd mention it.